source/source-connector/configuration-properties/change-stream.txt

```diff
@@ -86,6 +86,16 @@ Settings
          stream document. When set to ``true``, the connector
          automatically sets the ``change.stream.full.document`` property to
          ``updateLookup`` to receive the updated documents.
+       
+       .. note::
+          
+          This change stream option is a helper that returns the
+          ``fullDocument`` field from the full change stream document.
+          You must set pipeline configurations against the change stream
+          event, not the ``fullDocument``. To learn how to
+          configure a pipeline against a change stream event, see the
+          :ref:`Custom Pipeline <source-usage-example-custom-pipeline>` usage example.
```

Comment:
Issue:
A) I think this note tries to explain two different topics, but admonitions should address a single topic. The first sentence describes the purpose of the setting and the rest describes a requirement/behavior when specifying an aggregation pipeline for the change event.

B) I think "you must set pipeline configurations" is highly related to the "pipeline" setting and not this one.

Suggestion:

1) Update the description portion that starts with "Whether to publish the changed document" to incorporate the first sentence of your current admonition, e.g. (don't take this verbatim, just demonstrating one version):

"Whether to publish the fullDocument field from the change stream event. The fullDocument field contains the most current majority-committed version of the updated document. When set to true..."

2) Change the admonition to "important" since it presents an [essential point](https://www.mongodb.com/docs/meta/style-guide/style/callouts/).

3) I think the info about the pipeline setting could be relocated to the pipeline setting, and this admonition could be replaced with a mention that the reader should check that description on how to set that.